OPS-195: Procedural Tower Blurb (#68)
vatnz-dev/sops: docs/aerodromes/index.md

Pilots of VFR and IFR aircraft operating within Class D airspace must use a good lookout to separate themselves from each other as ATC separation is not provided. 

## Procedural 

> Procedural airspace is only found at a few airports around the country, theses are Towers without radar and rely on reports from pilots on thier position. 

Reviewer: ```suggestion
Procedural airspace is only found at a few airports around the country. These are Towers without radar and rely on reports from pilots on their position. 
```

Our procedural towers are: Gisborne, Napier, New Plymouth, Nelson, Dunedin and Invercargill. 

> They are all Class D therefore the above statement about Class D applies to them aswell. 

Reviewer: ```suggestion
They are all Class D therefore the all Class D rules apply to these aerodromes as well. 
```
